platforms/hyperledger-fabric/configuration/roles/create/ca-tools/tasks/main.yaml

```diff
@@ -2,18 +2,19 @@
 # This role creates value file for the deployment of CA Tools CLI
 #############################################################################################
 
-# Check if CA server is available created
+# Check if CA server is available
 - name: Check CA-server is available
   k8s_facts:
-    kind: Service
-    name: "{{ ca.name }}"
+    kind: Pod
     namespace: "{{ component_name }}"
     kubeconfig: "{{ kubernetes.config_file }}"
     context: "{{ kubernetes.context }}"
+    label_selectors:
+      - name = {{ component_services.ca.name }}
   register: get_ca
-  until: get_ca.resources|length > 0
-  retries: 20
-  delay: 40
+  until: ("Running" in get_ca|json_query('resources[*].status.phase'))
+  retries: "{{ network.env.retry_count}}"
```

Comment:
Kindly update the network.yaml sample file to state this additional variable entry